apps/users/forms.py

```diff
@@ -63,10 +63,11 @@ def __init__(self, *args, **kwargs):
 
     def save(self, request):
         user = super().save(request)
-        user.get_newsletters = self.cleaned_data["get_newsletters"]
-        user.language = get_language()
-        user.save()
-        return user
+        if user:
+            user.get_newsletters = self.cleaned_data["get_newsletters"]
+            user.language = get_language()
+            user.save()
+            return user
```

Comment:
Hm, would the return outside the if statement also work? It somehow looks strange to me, that in one case the function returns sth and the other not..